src/main/java/Event.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Represents a Task Object that is an event at a certain location.
+ */
+public class Event extends Task {
+    protected String at;

```

Comment:
maybe can try to use more discriptive words, instead of such short word, so that reader can understand what this variable repesents.. this is what i think